ci-operator/step-registry/ipi/conf/vsphere/usertags/ipi-conf-vsphere-usertags-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if [ -f ${SHARED_DIR}/tags_lists ]; then
```

Comment:
I think we do not need to judge `tags_lists` exists or not, because once this step is referenced in install workflow, then the provision step has to be called, or else it is some install workflow issue.

Reply:
ok